docs/api.md

```diff
@@ -720,6 +721,18 @@ Shortcut for [`keyboard.down`](#keyboarddownkey-options) and [`keyboard.up`](#ke
     - `omitBackground` <[boolean]> Hides default white background and allows capturing screenshots with transparency. Defaults to `false`.
 - returns: <[Promise]<[Buffer]>> Promise which resolves to buffer with captured screenshot
 
+#### page.select(selector, value)
+- `selector` <[string]> A [selector] to query page for
+- `value` <[string|string[]]> A string, or array strings in case of `multiple`, of options to select. If the provided value cannot be matched to the `option` value, it tries to match against the `option` label instead.
```

Comment:
It seems this should be `<[Array]<[string]>>` according to `args` format [here](https://github.com/GoogleChrome/puppeteer/blob/master/docs/api.md#puppeteerlaunchoptions).

Reply:
@vsemozhetbyt I wasn't sure and the doc linter didn't complain so I thought was okay. I'll push a fix, thanks.